app/api/src/players/players.service.ts

```diff
@@ -79,9 +79,9 @@ export class PlayersService {
 		let aggregatedShots = `SUM(${shotsValue})::integer`;
 		if (excludeDiscJumps) {
 			// Since subtracting disc jumps could theoretically drop the shots count
-			// to 0, clamp it to at least the number of hits, otherwise it'd be
-			// possible to have >100% accuracy.
-			aggregatedShots = `GREATEST(${aggregatedHits}, ${aggregatedShots} - ${aggregatedDiscJumps})`
+			// to 0, clamp it to at least the number of hits or 1, otherwise it'd be
+			// possible to divide by zero.
+			aggregatedShots = `GREATEST(1, ${aggregatedHits}, ${aggregatedShots} - ${aggregatedDiscJumps})`
```

Comment:
Added `1` to the list, because `aggregatedHits` could still be 0.

It's actually still possible for someone to have an accuracy >100%, because one shot can damage multiple targets. But if they pull that off in the totals, good for them. At least div by zero is fixed.